demo/chbench/dc.sh

```diff
@@ -226,11 +199,7 @@ Possible COMMANDs:
     `us demo-load \[ROPTS\]`         Generate a lot of changes to be used in the demo
 
  ROPTS (Run Options):
-     `uo --commit C`               Set both mz & peeker to 'unstable-C'
-     `uo --tag T`                  Set both mz & peeker to 'T'
-     `uo --mz-docker-tag T`        Set the tag for materialize/materialized:T
      `uo --mz-threads N`           Set mz -w/--threads to N
-     `uo --peeker-docker-tag T`    Set the tag for materialize/peeker:T
```

Comment:
This looks like it is no longer possible to build with a specific git sha or tag? I can't see where these things are added back.

Reply:
Ok, I added support for overriding mzcompose tags via env vars, and taught the implementation of these flags to set the new mzcompose env vars. I'd like to someday move support for this into mzcompose proper, but this seemed like the easiest way for now.